### PR TITLE
updated business logic

### PR DESCRIPTION
**Pull request by TohirjonIbnAlisher**

### CODE REVIEW

File: Task1/Program.cs [hunk 1]
```diff
@@ -6,7 +6,8 @@ static void Main(string[] args)
         {
             for(int i = 1;i < 100;i++)
             {
-                Console.WriteLine(i);
+                if (i % 2 == 0)
+                    Console.WriteLine(i);
```
Comment (zuxriddinCsharp) [inline Task1/Program.cs:10]: toq sonlar chiqsin

Comment (TohirjonIbnAlisher) [inline Task1/Program.cs:11]: toq

**Review by zuxriddinCsharp — CHANGES_REQUESTED** [review]

File: Task1/Program.cs (hunk 1 above)
Comment (TohirjonIbnAlisher) [reply Task1/Program.cs:11]: gdxfcvbnm.,

Comment (TohirjonIbnAlisher) [reply Task1/Program.cs:11]: fszdgxcfgvbjnmxhfxfxfcv

Comment (zuxriddinCsharp) [reply Task1/Program.cs:10]: kjgfhggrg

File: Task1/Program.cs [hunk 2]
```diff
@@ -8,6 +8,7 @@ static void Main(string[] args)
             {
                 if (i % 2 == 0)
                     Console.WriteLine(i);
+                // bu xatooooo
```
Comment (TohirjonIbnAlisher) [inline Task1/Program.cs:12]: akka xatooo

**Pull request merged**